frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1141,32 +1160,13 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
 
   const hasSearch = useMemo(() => columns.some((column) => column.search), [columns])
   const hasFilter = filters && filters.length > 0
-  const hasItems = (items && items.length > 0 && filtered) || nonZeroCount || indeterminateCount
-  const showToolbar = props.showToolbar !== false ? hasItems : false
+  const hasItems = items && items.length > 0 && filtered
+  const showToolbar = props.showToolbar !== false ? hasItems || emptyResult || loading : false
   const topToolbarStyle = items ? {} : { paddingBottom: 0 }
   const [isExportMenuOpen, setIsExportMenuOpen] = useState(false)
 
   const translatedPaginationTitles = usePaginationTitles()
 
-  const commonPaginationProps: Partial<Omit<PaginationProps, 'ref'>> = {
-    titles: translatedPaginationTitles,
-    itemCount,
-    toggleTemplate: indeterminateCount
-      ? ({ firstIndex, lastIndex }: { firstIndex?: number; lastIndex?: number }) => (
-          <>
-            <b>
-              {firstIndex} - {Math.min(lastIndex ?? 0, itemCount)}
-            </b>{' '}
-            {translatedPaginationTitles.ofWord} <b>{itemCount}+</b>
-          </>
-        )
-      : undefined,
-    perPage: perPage,
-    page: page,
-    onSetPage: (_event, page) => setPage(page),
-    onPerPageSelect: (_event, perPage) => updatePerPage(perPage),
-  }
-
```

Comment:
Some DRYing up here that could be preserved (with changes)

Reply:
DRY once again